.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: python
+
+branches:
+  only:
+    - master
+
+git:
+  depth: 10
+
+matrix:
+  include:
+    - os: linux
+      env: ATOM_CHANNEL=stable
+      before_install:
+        - pip install pyflakes
```

Comment:
You could move this step to the `'install:` stage, outside the matrix, and then leave any OS specific steps that need to happen before it in the `before_install:` step.